packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -55,6 +55,10 @@ const ControlsContainer = SpotlightContainerDecorator(
 
 // Keycode map for webOS TV
 const keyMap = {
+	'RED': 403,
+	'GREEN': 404,
+	'YELLOW': 405,
+	'BLUE': 406,
```

Comment:
The bonus to this approach is that its easier to test on the PC since you can emulate the keycode by mapping different keys using `core/keymap.add`

Reply:
`MoonstoneDecorator` adds those keys as `nonModal` and does not map each key separately. Do you suggest we add singular key/value pair?

Reply:
Ah! Didn't notice that when I saw them. That probably makes sense. Might be good (in a separate ticket) to support some indirection in keymap so that the `nonModal` block could refer to the names rather than the keycodes. Could get yourself caught in a loop if we didn't guard against it so maybe that's too "smart."